testdata/networking/ovn-egressfirewall/egressfirewall-policy2.yaml

```diff
@@ -0,0 +1,13 @@
+kind: EgressFirewall
+apiVersion: k8s.ovn.org/v1
+metadata:
+  name: default
+  namespace: test1
```

Comment:
`namespace` is empty for the other policies, should we remove it here?

Reply:
Sure, removed it.